docs/core/compatibility/core-libraries/7.0/max-precision-numeric-format-strings.md

```diff
@@ -0,0 +1,113 @@
+
+## Recommended action
+
+In most cases, no action is necessary, because it's unlikely that you're already using a precision higher than 999,999,999 in your format strings.
+
+## Affected APIs
+
+This change was implemented in the parsing logic that affects all numeric types.
+
+- <xref:System.Numerics.BigInteger.ToString(System.String)?displayProperty=fullName>
+- <xref:System.Numerics.BigInteger.ToString(System.String,System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.Numerics.BigInteger.TryFormat(System.Span{System.Char},System.Int32@,System.ReadOnlySpan{System.Char},System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.Int32.ToString(System.String)?displayProperty=fullName>
+- <xref:System.Int32.ToString(System.String,System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.Int32.TryFormat(System.Span{System.Char},System.Int32@,System.ReadOnlySpan{System.Char},System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.UInt32.ToString(System.String)?displayProperty=fullName>
+- <xref:System.UInt32.ToString(System.String,System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.UInt32.TryFormat(System.Span{System.Char},System.Int32@,System.ReadOnlySpan{System.Char},System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.Byte.ToString(System.String)?displayProperty=fullName>
+- <xref:System.Byte.ToString(System.String,System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.Byte.TryFormat(System.Span{System.Char},System.Int32@,System.ReadOnlySpan{System.Char},System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.SByte.ToString(System.String)?displayProperty=fullName>
+- <xref:System.SByte.ToString(System.String,System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.SByte.TryFormat(System.Span{System.Char},System.Int32@,System.ReadOnlySpan{System.Char},System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.Int16.ToString(System.String)?displayProperty=fullName>
+- <xref:System.Int16.ToString(System.String,System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.Int16.TryFormat(System.Span{System.Char},System.Int32@,System.ReadOnlySpan{System.Char},System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.UInt16.ToString(System.String)?displayProperty=fullName>
+- <xref:System.UInt16.ToString(System.String,System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.UInt16.TryFormat(System.Span{System.Char},System.Int32@,System.ReadOnlySpan{System.Char},System.IFormatProvider)?displayProperty=fullName>
+- <xref:System.Int64.Parse%2A?displayProperty=fullName>
+- <xref:System.Int64.TryParse%2A?displayProperty=fullName>
```

Comment:
I think these have to be updated to BigInteger if my other corrections are accurate.